mixins/localize-mixin.js

```diff
@@ -112,7 +112,13 @@ export const LocalizeMixin = superclass => class extends superclass {
 		}
 
 		const translatedMessage = new IntlMessageFormat(translatedValue, this.__language);
-		return translatedMessage.format(params);
+		let formattedMessage = translatedValue;
+		try {
+			formattedMessage = translatedMessage.format(params);
+		} catch (e) {
+			console.error(e);
```

Comment:
Does the resulting console statement contain enough info for us to track down what's going on?

Reply:
<img width="828" alt="Screen Shot 2020-02-14 at 5 42 39 PM" src="https://user-images.githubusercontent.com/5491151/74574355-77dae180-4f51-11ea-8a04-c701d4f30198.png">

Reply:
... based on the test, it  like it  👀 like does.